template/cordova/lib/PluginHandler.js

```diff
@@ -105,7 +105,7 @@ var handlers = {
             var type = obj.type;
 
             if(type === 'projectReference') {
-                project.removeProjectReference(path.join(plugin.dir, src), getTargetConditions(obj));
+                project.removeProjectReference(path.join('..', '..', 'plugins', plugin.id, src), getTargetConditions(obj));
```

Comment:
@daserge, please don't use hardcoded `../../plugins` path here. The idea here is that PlatformApi abstracts from cordova project structure so we ideally should not rely on presence of `plugins` directory two levels above. I'd rather calculate relative path from `plugin.dir` using `path.relative`